app/views/accounts/personal_keys/new.html.erb

```diff
@@ -0,0 +1,17 @@
+<% title t('account.personal_key.get_new') %>
+
+<h1 class="margin-top-0"><%= t('account.personal_key.get_new') %></h1>
+
+<p>
+  <%= t('account.personal_key.get_new_description') %>
+</p>
+
+<%= button_to(
+      create_new_personal_key_path,
+      method: 'post',
+      class: 'usa-button usa-button--wide usa-button--big margin-top-4',
```

Comment:
May want to double-check, but based on my related work in LG-4792, I've inferred that we tend to want 40px (2.5rem) around most of these primary action buttons, so I might bump that up here (particularly on the bottom). I think using `form_class` could be easiest to set a consistent y-margin and let margin collapse apply with the preceding paragraph's bottom margin and subsequent cancel link's top margin.

```suggestion
      class: 'usa-button usa-button--wide usa-button--big',
      form_class: 'margin-y-5',
```

Before|After
---|---
![image](https://user-images.githubusercontent.com/1779930/150349573-5f00f945-54d1-410c-94f6-152649b72ba7.png)|![image](https://user-images.githubusercontent.com/1779930/150349543-35cab1d0-4fbb-4490-bcaf-3cb3d2e6190a.png)

Reply:
Sure, I don't have as good of a sense about what these should be, so I'll trust you and accept the change, cc @nickttng to make sure it's on his radar

Reply:
The update looks good 👍🏼